mlrun/runtimes/nuclio/serving.py

```diff
@@ -496,10 +496,6 @@ def _add_ref_triggers(self):
                     if brokers:
                         brokers = brokers.split(",")
                     topic, brokers = parse_kafka_url(stream.path, brokers)
-                    max_workers_default = 4
```

Comment:
why not change default to 1?

Reply:
1 is the default in nuclio-jupyter. We could repeat it here if we want.